SnakeGame/view.py

```diff
@@ -1,14 +1,49 @@
 import pygame
+from model import CellContent
+import enum
 
 class GameView(object):
     
-    DEFAULT_SIZE = width, length = 540, 540
     BG_COLOR = green = 0, 135, 0
+    SNAKE_COLOUR = 135, 0, 0
+    FOOD_COLOUR = 255, 51, 0
+    SNAKEHEAD_COLOR = 0, 0, 0 
 
-    def __init__(self):
-        self.size = self.DEFAULT_SIZE
+    def __init__(self, gridDimensions=(10, 10), gridSize=(540, 540)):
+        #Grid dimensions in number of cells
+        self.width = gridDimensions[0]
+        self.height = gridDimensions[1]
+        #grid dimensions in pixels
+        self.gridSize=gridSize
+        #grid cell dimensions in pixels
+        self.cellWidth = gridSize[0]/self.width
+        self.cellHeight = gridSize[1]/self.height
 
-    def init(self):
-        self.screen = pygame.display.set_mode(self.size)
+    def render(self, grid):
+        self.screen = pygame.display.set_mode(self.gridSize)
         self.screen.fill(self.BG_COLOR)
-        pygame.display.flip()
+        
+        for i in range(self.height):
+            for j in range(self.width):
+                if(grid[i][j] == CellContent.Empty):
+                    pygame.draw.rect(self.screen, self.BG_COLOR, (self.getleftCoord(j,0), self.getTopCoord(i,0), self.cellWidth, self.cellHeight))
```

Comment:
I assume this is drawing a green rectangle for the cells that are empty? Would that be needed? The background is already green so not doing anything should work while saving us a drawing.

I replaced these two lines with:
```
if grid[i][j] == CellContent.Empty:
    continue
```
and it seemed to work as well.

A side note: parenthesis for if-conditions are not typically used in python unless it's a complex conditional. Again, not super important.